api/src/test/java/org/openmrs/ObsTest.java

```diff
@@ -165,14 +164,13 @@ public void shouldAddandRemoveObsToGroup() throws Exception {
 		// check duplicate add. should only be one
 		obsGroup.addGroupMember(obs);
 		assertTrue(obsGroup.hasGroupMembers(false));
-		assertEquals("Duplicate add should not increase the grouped obs size", 1, obsGroup.getGroupMembers().size());
+		assertEquals(obsGroup.getGroupMembers().size(), 1, "Duplicate add should not increase the grouped obs size");
```

Comment:
not sure why but the arguments should not be swapped here. the signature is https://junit.org/junit5/docs/5.0.1/api/org/junit/jupiter/api/Assertions.html#assertEquals-double-double- 

expected, then actual. in this case we expect the size to be 1

```suggestion
		assertEquals(1, obsGroup.getGroupMembers().size(), "Duplicate add should not increase the grouped obs size");
```

these things matter in case of error. otherwise we get a confusing message